### DISCUSSION TITLE
Images not showing when i Publish on Github

### DISCUSSION THREAD

**Discussion by OluwaseunHenry** [community/community#23896] [Projects and Issues]
<p>Hello Everyone!</p>
<p>I am having issues publishing on github. All my images are not showing after I publish, meanwhile they are displaying correctly on my local computer. I don’t know what’s cuasing this. Side note; this is my first try on a project using HTML and CSS, I am eager to share with others for review.</p>
<p>The Link to the project is:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/OluwaseunHenry/Order-Summary-Frontend-Mentor">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/OluwaseunHenry/Order-Summary-Frontend-Mentor" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/89458627/181079877-fc1d61f9-8ba0-4098-8758-657912270c00.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/2/62f9016f304e2f1fef04620aa80f2116619b911b_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/2/62f9016f304e2f1fef04620aa80f2116619b911b_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/2/62f9016f304e2f1fef04620aa80f2116619b911b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/2/62f9016f304e2f1fef04620aa80f2116619b911b_2_10x10.png"></div>

<h3><a href="https://github.com/OluwaseunHenry/Order-Summary-Frontend-Mentor" target="_blank" rel="noopener nofollow ugc">GitHub - OluwaseunHenry/Order-Summary-Frontend-Mentor</a></h3>

  <p>Contribute to OluwaseunHenry/Order-Summary-Frontend-Mentor development by creating an account on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by jfullstackdev** [comment, marked as answer]
<p>hello there,</p>
<p>you forget to include the folder path name <code>Order-Summary-Frontend-Mentor</code>,</p>
<p>say for example, one of your image is this,<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/6/76a55f6d6e620e8c39a117035532bf1758d962ad.png" data-download-href="/uploads/short-url/gVAEmxrk2z1fcKMALgpg8P5VgOF.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181079878-b2a56c64-0dde-4c9d-9835-516150539d81.png" alt="image" data-base62-sha1="gVAEmxrk2z1fcKMALgpg8P5VgOF" width="690" height="300" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/76a55f6d6e620e8c39a117035532bf1758d962ad_2_690x300.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/6/76a55f6d6e620e8c39a117035532bf1758d962ad_2_1035x450.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/6/76a55f6d6e620e8c39a117035532bf1758d962ad.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/76a55f6d6e620e8c39a117035532bf1758d962ad_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1077×469 41.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>so change all of your images path to:<br>
<a href="https://oluwaseunhenry.github.io/Order-Summary-Frontend-Mentor/images/%5Bname-of-the-image%5D" rel="noopener nofollow ugc">https://oluwaseunhenry.github.io/Order-Summary-Frontend-Mentor/images/[name-of-the-image]</a></p>

**Comment by airtower-luna** [comment]
<p>Either the above, or better use relative paths, not absolute ones. For example, change</p><aside class="onebox githubblob" data-onebox-src="https://github.com/OluwaseunHenry/Order-Summary-Frontend-Mentor/blob/bbe5c7c464707bda19676a04b1428358f2442035/index.html#L36">
  <header class="source">

      <a href="https://github.com/OluwaseunHenry/Order-Summary-Frontend-Mentor/blob/bbe5c7c464707bda19676a04b1428358f2442035/index.html#L36" target="_blank" rel="noopener">github.com</a>
  </header>

  <article class="onebox-body">
    <h4><a href="https://github.com/OluwaseunHenry/Order-Summary-Frontend-Mentor/blob/bbe5c7c464707bda19676a04b1428358f2442035/index.html#L36" target="_blank" rel="noopener">OluwaseunHenry/Order-Summary-Frontend-Mentor/blob/bbe5c7c464707bda19676a04b1428358f2442035/index.html#L36</a></h4>



  <pre class="onebox">    <code class="lang-html">
      <ol class="start lines" start="26" style="counter-reset: li-counter 25 ;">
          <li>      font-size: 11px;</li>
          <li>      text-align: center;</li>
          <li>    }</li>
          <li>    .attribution a {</li>
          <li>      color: hsl(228, 45%, 44%);</li>
          <li>    }</li>
          <li>  &lt;/style&gt;</li>
          <li>&lt;/head&gt;</li>
          <li>&lt;body&gt;</li>
          <li>  &lt;div class="main"&gt;</li>
          <li class="selected">    &lt;img src="/images/illustration-hero.svg" alt="" /&gt;</li>
          <li>
          </li>
<li>    &lt;div class="order"&gt;</li>
          <li>      &lt;h1&gt;Order Summary&lt;/h1&gt;</li>
          <li>
          </li>
<li>      &lt;p&gt;</li>
          <li>        You can now listen to millions of songs, audiobooks, and podcasts on</li>
          <li>        any device anywhere you like!</li>
          <li>      &lt;/p&gt;</li>
          <li>    &lt;/div&gt;</li>
          <li>    &lt;div class="annual-plan"&gt;</li>
      </ol>
    </code>
  </pre>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<p>
to:</p>
<pre><code class="lang-auto">      &lt;img src="images/illustration-hero.svg" alt="" /&gt;
</code></pre>
<p>Note the lack of starting slash. That way the browser will use that path relative to the directory the current page is in.</p>
